src/util/rop_rrr.jl

```diff
@@ -0,0 +1,267 @@
+
+
+function run_rrr(network, model_constructor, optimizer; time_limit::Float64=3600.0, kwargs...)
```

Comment:
I was a little surprised there were not more parameters to manage here than just the time limit.  Might think if there are some more algorithm settings that the user might like to be able to change.

Reply:
added recovery problem time limit, and minimum sub problem time limit as kwargs